src/meta/topics/dependencies.md

```diff
@@ -0,0 +1,90 @@
+# Dependencies
+
+There are several dependencies that Node.js relies on to work the way it does.
+
+- [Libraries](#Libraries)
+	- [V8](#V8)
+	- [libuv](#libuv)
+	- [http-parser](#http-parser)
+	- [c-ares](#c-ares)
+	- [OpenSSL](#OpenSSL)
+	- [zlib](#zlib)
+- [Tools](#Tools)
+	- [npm](#npm)
+	- [gyp](#gyp)
+	- [gtest](#gtest)
```

Comment:
editorconfig seems to be the best option for this. This is yet another topic that has been gone over extensively by the Website WG - I think trying to pull some of them in (@nodejs/website) to share the experience they've had and conclusions the WG has come to would be a good idea.